website_and_docs/content/documentation/grid/configuration/toml_options.ja.md

```diff
@@ -113,9 +116,13 @@ max-sessions = 2
 
 [docker]
 configs = [
-        "selenium/standalone-chrome:93.0", "{\"browserName\": \"chrome\", \"browserVersion\": \"91\"}", 
-        "selenium/standalone-firefox:92.0", "{\"browserName\": \"firefox\", \"browserVersion\": \"92\"}"
-    ]
+    "selenium/standalone-chrome:93.0", "{\"browserName\": \"chrome\", \"browserVersion\": \"91\"}", 
+    "selenium/standalone-firefox:92.0", "{\"browserName\": \"firefox\", \"browserVersion\": \"92\"}"
+]
+#Optionally define all device files that should be mapped to docker containers
```

Comment:
Can we please leave this commented out? People tend to copy & paste and this will fail in Windows or other systems where `/dev/kvm` is not available.

Reply:
That makes sense. I'll be working on it

Reply:
@diemol, Would it be nice if I just put an empty array like this? 
`devices = [ ]`

Reply:
Better all commented out
```
#devices = [
#    "/dev/kvm:/dev/kvm"
#]
```

Reply:
@diemol, in the last commit this issue was resolved.